<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegDialog</class>
 <widget class="QDialog" name="RegDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/res/gdfgsdfhsdnh.jpeg);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>216</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>确认密码</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="usernameLineEdit">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="confirmPasswordLineEdit">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="registerButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>sgin up</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
